tests/ai/src/tool-calling.ts

```diff
@@ -0,0 +1,121 @@
+import { ChatPrompt, IChatModel } from '@microsoft/teams.ai';
+import { ActivityLike, IMessageActivity, SentActivity } from '@microsoft/teams.api';
+
+interface PokemonSearch {
+  pokemonName: string;
+}
+
+export const handlePokemonToolCalling = async (
+  model: IChatModel,
+  activity: IMessageActivity,
+  send: (activity: ActivityLike) => Promise<SentActivity>
+) => {
+  // :snippet-start: single-function-calling
+  // activity could have text like 'pikachu'
```

Comment:
Should this comment be above the snippet?

Reply:
Moved it closer to where it's being called